### ISSUE TITLE
dorado duplex data analysis

### ISSUE THREAD

**Issue by HenkvdMeulen** [nanoporetech/dorado#261]
Dear Nanopore,
I have been trying to analyze my raw reads from a MinION R10.4.1 flow cell with Dorado Duplex basecalling. I converted the fast5 files to pod5, then ran this command:

dorado duplex dna_r10.4.1_e8.2_400bps_sup@v4.1.0 pod5files/ > duplex.bam

the directory contains multi pod5 files, which in total contain about 1.5M reads of 1Gbps of data. it took about 2 hours to perform this analysis with an NVIDIA RTX A4500 gpu.

I converted this bam file to fast q, first using samtools sort to create a sorted bam file, and then bam2fq to convert it to a fastq file. I would like to know if so far, this would be a good approach.

When I checked the quality of the fastq file using FastQC, the q-score of the reads was close to q5 instead of q20+ that I'm looking for. In addition, fastqc somehow seems to concatinate all the reads, instead of overlapping the locations of the bases (see fastqc report attached)

[bam2fq_fastqc.zip](https://github.com/nanoporetech/dorado/files/11865388/bam2fq_fastqc.zip)

In short, could you tell me if I produce the bam-file in the correct way and how to produce a high quality fastq file from that? thanks in advance!

**Comment by vellamike** [comment]
Hi @HenkvdMeulen , it appears that nobody replied to your question, this was an oversight for which I apologise. 

This appears to be more of a FastQC question, which is a tool we don't generally recommend for use with nanopore long reads. Dorado duplex has also advanced significantly since your question and I suspect this question is no longer relevant so I'm closing it. If however you still need help please request to reopen.
